core/shared/src/main/scala/scalaz/zio/IO.scala

```diff
@@ -0,0 +1,357 @@
+
+  /** See [[scalaz.zio.ZIO.fromFunctionM]]
+   */
+  final def fromFunctionM[E, A](f: Any => IO[E, A]): IO[E, A] =
+    ZIO.fromFunctionM(f)
+
+  /** See [[scalaz.zio.ZIO.fromFuture]]
+   */
+  final def fromFuture[A](make: ExecutionContext => scala.concurrent.Future[A]): Task[A] =
+    ZIO.fromFuture(make)
+
+  /** See [[scalaz.zio.ZIO.fromOption]]
+   */
+  final def fromOption[A](v: => Option[A]): IO[Unit, A] = ZIO.fromOption(v)
+
+  /** See [[scalaz.zio.ZIO.fromTry]]
+   */
+  final def fromTry[A](value: => scala.util.Try[A]): Task[A] =
+    ZIO.fromTry(value)
+
+  /** See [[scalaz.zio.ZIO.halt]]
+   */
+  final def halt[E](cause: Cause[E]): IO[E, Nothing] = ZIO.halt(cause)
+
+  /** See [[scalaz.zio.ZIO.identity]]
+   */
+  final def identity: IO[Nothing, Any] = ZIO.identity
+
+  /** See [[scalaz.zio.ZIO.interruptible]]
+   */
+  final def interruptible[E, A](zio: IO[E, A]): IO[E, A] =
+    ZIO.interruptible(zio)
+
+  /** See [[scalaz.zio.ZIO.interruptibleMask]]
+   */
+  final def interruptibleMask[E, A](k: ZIO.InterruptStatusRestore => IO[E, A]): IO[E, A] =
+    ZIO.interruptibleMask(k)
+
+  /** See [[scalaz.zio.ZIO.lock]]
+   */
+  final def lock[R, E, A](executor: Executor)(zio: IO[E, A]): ZIO[R, E, A] =
+    ZIO.lock(executor)(zio)
+
+  /** See [[scalaz.zio.ZIO.mergeAll]]
+   */
+  final def mergeAll[E, A, B](in: Iterable[IO[E, A]])(zero: B)(f: (B, A) => B): IO[E, B] =
+    ZIO.mergeAll(in)(zero)(f)
+
+  /** See [[scalaz.zio.ZIO.mergeAllPar]]
+   */
+  final def mergeAllPar[E, A, B](in: Iterable[IO[E, A]])(zero: B)(f: (B, A) => B): IO[E, B] =
+    ZIO.mergeAllPar(in)(zero)(f)
+
+  /** See [[scalaz.zio.ZIO.raceAll]]
+   */
+  final def raceAll[E, A](zio: IO[E, A], ios: Iterable[IO[E, A]]): IO[E, A] = ZIO.raceAll(zio, ios)
+
+  /** See [[scalaz.zio.ZIO.reduceAll]]
+   */
+  final def reduceAll[E, A](a: IO[E, A], as: Iterable[IO[E, A]])(f: (A, A) => A): IO[E, A] =
+    ZIO.reduceAll(a, as)(f)
+
+  /** See [[scalaz.zio.ZIO.reduceAllPar]]
+   */
+  final def reduceAllPar[E, A](a: IO[E, A], as: Iterable[IO[E, A]])(f: (A, A) => A): IO[E, A] =
+    ZIO.reduceAllPar(a, as)(f)
+
+  /** See [[scalaz.zio.ZIO.require]]
+   */
+  final def require[E, A](error: E): IO[E, Option[A]] => IO[E, A] =
+    ZIO.require(error)
+
+  /** See [[scalaz.zio.ZIO.reserve]]
+   */
+  def reserve[E, A, B](reservation: IO[E, Reservation[Any, E, A]])(use: A => IO[E, B]): IO[E, B] =
+    ZIO.reserve(reservation)(use)
+
+  /** See [[scalaz.zio.ZIO.runtime]]
+   */
+  final def runtime: IO[Nothing, Runtime[Any]] = ZIO.runtime
+
+  /** See [[scalaz.zio.ZIO.succeed]]
+   */
+  final def succeed[A](a: A): UIO[A] = ZIO.succeed(a)
+
+  /** See [[scalaz.zio.ZIO.succeedLazy]]
+   */
+  final def succeedLazy[A](a: => A): UIO[A] = ZIO.succeedLazy(a)
+
+  /** See [[scalaz.zio.ZIO.supervise]]
+   */
+  final def supervise[E, A](zio: IO[E, A]): IO[E, A] =
+    ZIO.supervise(zio)
+
+  /** See [[scalaz.zio.ZIO.supervised]]
+   */
+  def supervised[E, A](zio: IO[E, A]): IO[E, A] =
```

Comment:
It would be good to "search and replace" all `zio` parameter names with `io` parameter names. 👍